pr-deploy/README.md

```diff
@@ -0,0 +1,55 @@
+AMP PR Deploy Bot
+===================
+
+A GitHub App deploys a PR branch to a static website when you ask it to. 
+
+Websites are served by a [Google Cloud Storage bucket](https://console.cloud.google.com/storage/browser/amp-test-website-1). <br>
+The app is hosted by [Google App Engine](https://amp-pr-deploy-bot.appspot.com) and is installed exclusively on the ampproject/amphtml repository.
+
+How the app works
+----------------
+1. A commit is pushed to a new or existing pull request on ampproject/amphtml.
+2. Travis compiles your changes and uploads the build artifacts and example pages as `amp_dist_<Travis build number>.zip` to a remote storage location. During this step, a check called `ampproject/pr-deploy` is set to `pending`.
+3. a) If there was a compilation error, Travis tells the AMP PR Deploy Bot that there's nothing left to do until the error is fixed. <br>
+   b) If there were no errors, Travis tells the AMP PR Deploy Bot that a test site is ready to be deployed. `ampproject/pr-deploy` is now `netural`
+4. A test site is deployed by clicking the 'Deploy Me' button in the details page of `ampproject/pr-deploy`. The app unzips and writes `amp_dist_<Travis build number>.zip` to the public Google Cloud Storage bucket.
+5. `ampproject/pr-deploy` completes with the website URL. `https://console.cloud.`
+
+Here's a quick [demo](https://github.com/ampproject/amphtml/pull/24274) on how to create a test site.
+
+Local development
+-----------------
+
+1. Clone this repository and cd into the `pr-deploy` directory.
+2. `npm install`
+3. Start a new [Smee channel](https://smee.io/). This can be used to proxy
+   GitHub webhooks to your local machine.
+5. Create a new [GitHub App](https://github.com/settings/apps/new) with the following settings:
+   * General
+     * Set _Homepage URL_ to the Smee channel
+     * Set _Webhook URL_ to the Smee channel
+     * Set _Webhook Secret_ to any pin your choice
+   * Permissions and Events
+     * Set _Checks_ to Read & write
+     * Set _Metadata_ to Read-only
+     * Set _Pull requests_ to Read-only
+     * Subscribe to events: _Check run_, _Pull request_   
+6. After creating the application, generate and download a private key. Also
+   take note of the App ID.
+7. Create a personal access token belonging to a GitHub _user_ with the
+   `public_repo` and `read:org` permissions and nore its access token.
+8. Install the application on a GitHub repository that you want to use for
+   testing. You might want to create a new repository for this purpose.
```

Comment:
"You might want to fork the [ampproject/amphtml](https://github.com/ampproject/amphtml) repository or create a new one for this purpose"